libnmstate/schemas/operational-state.yaml

```diff
@@ -480,6 +481,30 @@ definitions:
                     type: boolean
                   state:
                     type: string
+  interface-team:
+    rw:
+      properties:
+        type:
+          type: string
+          enum:
+          - team
+        team:
+          type: object
+          properties:
+            ports:
+              type: array
+              items:
+                type: object
+                properties:
+                  name:
+                    type: string
+            runner:
+              type: object
+              properties:
+                name:
+                  type: string
+                  enum:
+                  - loadbalance
```

Comment:
I see no reason to limit it in the schema.

Reply:
I am fine with removing this for now. I am preparing a little patch to discuss about this (this patch is not going to break API compatibility).

Thanks!